delegation-toolkit/how-to/create-smart-account.md

```diff
@@ -157,15 +142,12 @@ export const walletClient = createWalletClient({
 </TabItem>
 </Tabs>
 
-### Configure a WebAuthn (passkey) signatory
+### Create a Hybrid smart account with a WebAuthn (passkey) signatory
 
-This example creates a [Viem WebAuthn Account](https://viem.sh/account-abstraction/accounts/webauthn) as the signatory,
-using Viem's `toWebAuthnAccount` function.
+Use [`toMetaMaskSmartAccount`](../reference/api/smart-account.md#tometamasksmartaccount) and Viem's [`toWebAuthnAccount`](https://viem.sh/account-abstraction/accounts/webauthn) to create a Hybrid smart account with a WebAuthn Account signatory:
 
 :::info Installation required
-
-To work with WebAuthn, install the [Ox SDK](https://oxlib.sh/).
-
+To use WebAuthn, install the [Ox SDK](https://oxlib.sh/).
```

Comment:
We can revert, because WebAuthn is part of Viem SDK, but 0x is required for getting address, and PublicKey from credentials.